dev/breeze/src/airflow_breeze/prod/build_prod_image.py

```diff
@@ -0,0 +1,196 @@
+    "build_id",
+    "commit_sha",
+    "airflow_image_readme_url",
+    "install_providers_from_sources",
+    "install_from_pypi",
+    "install_from_docker_context_files",
+]
+
+PARAMS_TO_VERIFY_PROD_IMAGE = [
+    "dev_apt_command",
+    "dev_apt_deps",
+    "runtime_apt_command",
+    "runtime_apt_deps",
+]
+
+
+def construct_arguments_docker_command(prod_image: ProdParams) -> List[str]:
+    args_command = []
+    for param in PARAMS_PROD_IMAGE:
+        args_command.append("--build-arg")
+        args_command.append(param.upper() + "=" + str(getattr(prod_image, param)))
+    for verify_param in PARAMS_TO_VERIFY_PROD_IMAGE:
+        param_value = str(getattr(prod_image, verify_param))
+        if len(param_value) > 0:
+            args_command.append("--build-arg")
+            args_command.append(verify_param.upper() + "=" + param_value)
+    docker_cache = prod_image.docker_cache_prod_directive
+    if len(docker_cache) > 0:
+        args_command.extend(prod_image.docker_cache_prod_directive)
+    return args_command
+
+
+def construct_docker_command(prod_image: ProdParams) -> List[str]:
+    arguments = construct_arguments_docker_command(prod_image)
+    build_command = prod_image.check_buildx_plugin_build_command()
+    build_flags = prod_image.extra_docker_build_flags
+    final_command = []
+    final_command.extend(["docker"])
+    final_command.extend(build_command)
+    final_command.extend(build_flags)
+    final_command.extend(["--pull"])
+    final_command.extend(arguments)
+    final_command.extend(["-t", prod_image.airflow_prod_image_name, "--target", "main", "."])
+    final_command.extend(["-f", 'Dockerfile'])
+    final_command.extend(["--platform", prod_image.platform])
+    return final_command
+
+
+def login_to_docker_registry(build_params: ProdParams):
+    if build_params.ci == "true":
+        if len(build_params.github_token) == 0:
+            console.print("\n[blue]Skip logging in to GitHub Registry. No Token available!")
+        elif build_params.airflow_login_to_github_registry != "true":
+            console.print(
+                "\n[blue]Skip logging in to GitHub Registry.\
+                    AIRFLOW_LOGIN_TO_GITHUB_REGISTRY is set as false"
+            )
+        elif len(build_params.github_token) > 0:
+            run_command(['docker', 'logout', 'ghcr.io'], verbose=True, text=True)
+            run_command(
+                [
+                    'docker',
+                    'login',
+                    '--username',
+                    build_params.github_username,
+                    '--password-stdin',
+                    'ghcr.io',
+                ],
+                verbose=True,
+                text=True,
+                input=build_params.github_token,
+            )
+        else:
+            console.print('\n[blue]Skip Login to GitHub Container Registry as token is missing')
+
+
+def clean_docker_context_files():
+    extensions_to_delete = ['whl', 'tar.gz']
```

Comment:
I think we should just skip README.md. There might be other files in §`docker-context-files` that might pollute the cache.

Reply:
ok @potiuk I will these changes.

Reply:
@potiuk I fixed this change